libs/utils/decorate.js

```diff
@@ -228,12 +228,80 @@ function getVideoAttrs(hash, dataset) {
   return `${globalAttrs} controls`;
 }
 
+export function syncPausePlayIcon(video) {
+  const playIcon = video.nextElementSibling?.querySelector('.play-icon');
+  const pauseIcon = video.nextElementSibling?.querySelector('.pause-icon');
+  if (video.paused || video.ended) {
+    playIcon?.classList.remove('hidden');
+    pauseIcon?.classList.add('hidden');
+  } else {
+    pauseIcon?.classList.remove('hidden');
+    playIcon?.classList.add('hidden');
+  }
+}
+
+export function addAccessibilityControl(videoString, videoAttributes, tabIndex = 0) {
+  const labels = {
+    accessibilityLabel: 'play/pause motion',
+    playLabel: 'play icon',
+    pauseLabel: 'pause icon'
+  }
+  if (!videoAttributes.includes('controls')) {
+    if (videoAttributes.includes('hoverplay')) {
+      return `<a class='pause-play-wrapper video-holder' tabindex=${tabIndex} aria-label='${labels.accessibilityLabel}' >${videoString}
+    </a>`;
+    }
+    return `<div class='video-container video-holder'>${videoString}
+    <a class='pause-play-wrapper' role='button' tabindex=${tabIndex} aria-label=' ${labels.accessibilityLabel}'>
+      <div class='offset-filler'>  
+        <img class='accessibility-control pause-icon ${videoAttributes.includes('autoplay') ? '' : 'hidden'}' alt='${labels.pauseLabel}' src='https://main--federal--adobecom.hlx.page/federal/assets/svgs/accessibility-pause.svg'/>
+        <img class='accessibility-control play-icon ${videoAttributes.includes('autoplay') ? 'hidden' : ''}' alt='${labels.playLabel}' src='https://main--federal--adobecom.hlx.page/federal/assets/svgs/accessibility-play.svg'/>
```

Comment:
I think we might still need to use actual placeholders for this, defined in the placeholders sheet and have them translated, so they remain relevant on all locales.